src/main/java/service/impl/ConverterToFruitTransactionImpl.java

```diff
@@ -0,0 +1,33 @@
+package service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import model.Operation;
+import service.ConvertToFruitTransactionService;
+
+public class ConverterToFruitTransactionImpl implements ConvertToFruitTransactionService {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convert(List<String> rawData) {
+        if (rawData == null) {
+            throw new RuntimeException("Incorrect data passed!");
+        }
```

Comment:
No need to add null checks all the time. This clutters up the business logic and makes the code difficult to understand
```suggestion
```